configuration/buildspec.yml

```diff
@@ -0,0 +1,37 @@
+version: 0.2
+phases:
+  install:
+    runtime-versions:
+      nodejs: latest
+      dotnet: latest
+    commands:
+      - /usr/local/bin/dotnet-install.sh --channel LTS      
+  pre_build:
+    commands:
+      - echo Pre Build started on `date`
+      - cd Buddies.Web/
+      - yarn install
+      - yarn run lint -- --fix
+      - cd ..
+      - dotnet new tool-manifest --force
+      - dotnet tool install Swashbuckle.AspNetCore.Cli
```

Comment:
Have you tried using `dotnet tool restore` here rather than creating a new `dotnet-tools.json`? I included one in the repo per recommendation from the .NET SDK docs and so that we can be consistent when getting CLI tools.

Reply:
that works, fixed.